dietpi/dietpi-software

```diff
@@ -1880,6 +1880,14 @@ _EOF_
 		aSOFTWARE_CATEGORY_INDEX[$software_id]=18
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='https://dietpi.com/docs/software/printing/#octoprint'
 		(( $G_HW_ARCH < 3 && $G_DISTRO > 4 )) || aSOFTWARE_REQUIRES_BUILDESSENTIAL[$software_id]=1
+		#------------------
+		software_id=187
+
+		aSOFTWARE_NAME[$software_id]='CUPS'
+		aSOFTWARE_DESC[$software_id]='common UNIX Pprinting system'
```

Comment:
Typo: `Pprinting` => `printing`

Reply:
will be updated once we agreed on installation procedure